test/bdd/pkg/chs/openapi/client/confidential_storage_hub_client.go

```diff
@@ -0,0 +1,111 @@
+// Code generated by go-swagger; DO NOT EDIT.
```

Comment:
Please move it under pkg/client https://github.com/trustbloc/edge-service/tree/main/pkg/client/csh

Reply:
other components like comparator need to use the client